tmux-onedark-theme.tmux

```diff
@@ -12,6 +12,26 @@ get_widgets() {
    echo "$(tmux show-option -gqv "@onedark_widgets")"
 }
 
+get_variable() {
+  echo "$(tmux show-option -gqv "@onedark_$1")"
+}
+
+get_time_string() {
+  if [ "$(get_variable "twelve_hour")" == "on" ]; then
+    echo "%l:%M %p"
+  else
+    echo "%H:%M"
+  fi
+}
+
+get_date_string() {
+  if [ "$(get_variable "month_first")" == "on" ]; then
```

Comment:
I was actually thinking the same thing, I considered changing it but decided to wait on your feedback first.